src/extra/signal/_ftd.pyx

```diff
@@ -19,14 +19,51 @@ import numpy as np
 ctypedef floating data_t
 ctypedef floating[::contiguous] data_array_t
 
+
 cpdef enum EdgeInterpolation:
-    # Interpolation method to find edge position.
+    """Edge interpolation methods.
+
+    The fast timing discriminators in this module can use several
+    different interpolation methods to estimate the real edge position
+    in between the actual samples. These differ in terms of their
+    interpolation quality and performance requirements.
+
+    The default setting is to apply `EdgeInterpolation.LINEAR`, which
+    offers good results for not-too-fast rise times at close to no
+    cost in performance.
+
+    For particularly fast signals however, i.e. with large curvature,
+    linear interpolation tends to bias edge positions towards the sample
+    positions at either end. This can be improved with
+    `EdgeInterpolation.SINC`, but at a significant performance hit.
+
+    If the discriminated edge positions should always fall on the most
+    actual sample positions, `EdgeInterpolation.NEAREST` chooses the
+    closest such position. Note that this offers no relevant performance
+    advantage over linear interpolation.
+
+    Some of the interpolation methods can be configured via
+    [config_ftd_interpolation][extra.signal.config_ftd_interpolation].
+
+    Attributes:
+        NEAREST: Chose the sample position whose value is closest to the
+            value at the true edge position, no cost in performance and
+            will always return an integer result.
+        LINEAR: Perform a linear interpolation between the two bordering
+            samples to find the position.
+        SPLINE: NYI.
```

Comment:
Nitpicking, I'd use `Not yet implemented` in full here to avoid confusion.

Reply:
You're right. On second thought though, I'm not sure why include it in the first place. It's an enum anyway, so we can add it later any time.